modules/create-resource-init-bundle.adoc

```diff
@@ -85,8 +88,35 @@ $ kubectl create -f <init_bundle>.yaml \ <2>
 <1> Create the project where secured cluster resources will be installed. This example uses `stackrox`.
 <2> Specify the file name of the init bundle containing the secrets.
 <3> Specify the project name that you created. This example uses `stackrox`.
-endif::[]
-
+endif::kube[]
 
 .Next Step
 * Install {product-title-short} secured cluster services in all clusters that you want to monitor.
+
+ifeval::["{context}" == "init-bundle-cloud-ocp"]
+:!openshift:
```

Comment:
I don't really understand the syntax but could you please explain why this is with exclamation mark?
I have noticed that sometimes such blocks are the same as in the begining (e.g. https://github.com/openshift/openshift-docs/blob/4c928284d0f9c9cc2919d2e96d7bcaf3b11e4c1d/modules/deploy-app.adoc?plain=1#L11 )